public/sass/pages/_alerting.scss

```diff
@@ -132,6 +132,13 @@
   }
 }
 
+.no-alert {
```

Comment:
this should be in the _panel_alertlist.scss no? And maybe named something more specific

Reply:
All the other alert-list classes are in the _alerting.scss. Maybe they should all be in the _panel_alertlist.scss? Will rename class.